build/ProjectReunion-BuildFoundation.yml

```diff
@@ -148,6 +148,39 @@ jobs:
       PathtoPublish: '$(build.SourcesDirectory)\BuildOutput'
       artifactName: 'BuildOutput'
 
+  - task: PowerShell@2
```

Comment:
I think this should maybe go into
AzurePipelinesTemplates\WindowsAppSDK-SetupBuildEnvironment-Steps.yml